src/image_occlusion_enhanced/ngen.py

```diff
@@ -504,6 +506,7 @@ def _saveMaskAndReturnNote(self, omask_path, qmask, amask, img, note_id, nid=Non
             mw.col.addNote(note)
             logger.debug("!notecreate %s", note)
 
+        return note
```

Comment:
don't ask me why a method called `_saveMaskAndReturnNote` never returned a note in the first place :man_shrugging: